.github/workflows/ci.yml

```diff
@@ -75,3 +75,39 @@ jobs:
           path: |
             target/scoverage-report/**/*
             target/test-reports/**/*
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: sbt-build-test
+    if: ${{ github.ref == 'refs/heads/pg-store' }}
```

Comment:
Do we intend to merge this into non-pg-store?

It may be cleaner to let the worker do some double work (i.e. build it twice) and have another pipeline that only runs for commits on specific branches (and for tags) that builds the container.

Reply:
For now we'll keep it here, we can iterate on it after the `pg-store` branch finally makes it to `main`.